day_6/exercises/burrito.js

```diff
@@ -16,12 +16,25 @@ class Burrito {
     this.base = base;
     this.toppings = toppings;
   }
+
+  addTopping() {
+    console.log(this.toppings + " peppers")
```

Comment:
Can you make this method dynamic? How would you utilize a parameter to add a topping to the toppings array?